### ISSUE TITLE
Height difference threshold option?

### ISSUE THREAD

**Issue by BdNp** [garris/BackstopJS#296]
It would be useful to allow for the height difference to compare against a threshold of its own. The data content in my company's project varies differently between our production and QA environments and constantly fails because of this.

**Comment by garris** [comment]
Sure,  this has come up before.   Couple of questions...

I guess a property which accepts pixels is sufficient?

Does the 'removeSelectors' not suffice in this context?  

Is there a reason why do don't us stubs or replace content in the element you're concerned about? 

Did you know you could use a BackstopJS onReady script to change content such that you would always test the expected behavior?

**Comment by BdNp** [comment]
Yeah, a pixel acceptance property would be enough. 'removeSelectors' works in some cases but not all. And yes, my next step would be to replace and normalize the content in an onReady script. A height property threshold would be more of a time-saver.

**Comment by garris** [comment]
Just curious, if you get a minute, what is the case where removeSelectors doesn't work?

**Comment by BdNp** [comment]
Well I wasn't sure until this morning. Turns out one of our developers had applied padding-top to the body tag, which is what is throwing everything off. Of course. removing the body selector renders the entire test empty. 

The truth is, this would be a perfect case of the tests failing regression. I still think a pixel height threshold would be useful, especially if the entire team is expecting a global change like this. But in this case, I should be able to remove the offending class for the sake of testing smaller elements.
